From: David Faure <faure@kde.org>
Date: Fri, 03 Jan 2014 12:39:15 +0000
Subject: adjust for new (= old) kdnssd header prefix
X-Git-Url: http://quickgit.kde.org/?p=plasma-framework.git&a=commitdiff&h=e577f4e588d427d77da2446c41b00566e860a35d
---
adjust for new (= old) kdnssd header prefix

Make sure to update kdnssd-framework first.
---


--- a/src/plasma/private/service_p.h
+++ b/src/plasma/private/service_p.h
@@ -31,8 +31,8 @@
 #include <kplugininfo.h>
 #include <klocalizedstring.h>
 
-#include <kdnssd/publicservice.h>
-#include <kdnssd/servicebrowser.h>
+#include <dnssd/publicservice.h>
+#include <dnssd/servicebrowser.h>
 
 #include "plasma/configloader.h"
 

--- a/src/plasma/service.cpp
+++ b/src/plasma/service.cpp
@@ -31,8 +31,8 @@
 #include <kservicetypetrader.h>
 #include <ksharedconfig.h>
 
-#include <kdnssd/publicservice.h>
-#include <kdnssd/servicebrowser.h>
+#include <dnssd/publicservice.h>
+#include <dnssd/servicebrowser.h>
 #include <qstandardpaths.h>
 
 #include "configloader.h"

